examples/reduce-counter/Cargo.toml

```diff
@@ -10,4 +10,5 @@ path = "src/main.rs"
 [dependencies]
 tonic = "0.11.0"
 tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
-numaflow-rs = { path = "../../" }
+numaflow = { path = "../../" }
```

Comment:
Yep, i agree i just went with `numaflow`, since the root level cargo.toml defined the package as `numaflow`. Also note that cargo does not allow library names to contain hyphens so we would need to do `numaflow_rs`. If Vigith and Yashash are ok with the change then ill rename all the instances of the package with numaflow_rs

Reply:
let's use `numaflow`, we do not need the `-rs`.

Reply:
> Crate names should not use `-rs` or `-rust` as a suffix or prefix. Every crate is Rust! It serves no purpose to remind users of this constantly.

https://rust-lang.github.io/api-guidelines/naming.html#:~:text=Crate%20names%20should%20not%20use,remind%20users%20of%20this%20constantly.

Reply:
Nice finding!